service-api/module/Application/src/Controller/IdentityController.php

```diff
@@ -69,11 +70,10 @@ public function createAction(): JsonModel
 
         $this->getResponse()->setStatusCode(Response::STATUS_CODE_400);
 
-        return new JsonModel([
-            'type' => 'HTTP400',
-            'title'  => 'Invalid data',
-            'error' => $validator->getMessages(),
-        ]);
+        return new JsonModel(new Problem(
+            'Invalid data',
```

Comment:
Do we not need to explicitly specify this as a title?

Reply:
Since it's the first argument in the constructor it knows it's the title. `extra` only needs to be specified because we're skipping the other arguments